app/src/main/java/com/example/caculator/MainActivity.kt

```diff
@@ -12,5 +12,69 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        updateUI("")
     }
+
+    val opList: MutableList<String> = arrayListOf()
```

Comment:
move defind variable on top of file. convention name of variable: example: opList -> mOpList for global variable